Makefile

```diff
@@ -32,7 +32,7 @@ test-main: tidy
 
 
 .PHONY: test
-test: test-unit test-main
+test: test-unit test-integration test-main
```

Comment:
Should I have added this to `make pre-pr` instead? Either way, it's easy to forget about integration testing right now since it is not in `make test` or `make pre-pr`

Reply:
Either way - there is a card to replace testing of the executable, so that we can use some go tooling for it.